exosphere-shared/templates/add-service/exoservice-ls-mongodb/features/support/world.ls

```diff
@@ -6,7 +6,7 @@ require! {
 World = !->
 
   # Fills in _____modelName_____ ids in the placeholders of the template
-  @fill-in-_____modelName_____-ids = (template, done) ->
+  @fill-in-modelName-ids = (template, done) ->
```

Comment:
You shouldn't be running the tests of these folders without those lines being replaced. They are placeholders that get converted when a service gets scaffolded